package.json

```diff
@@ -251,6 +251,7 @@
     "@metamask/permission-controller": "^4.0.0",
     "@metamask/phishing-controller": "^3.0.0",
     "@metamask/post-message-stream": "^6.0.0",
+    "@metamask/ppom-validator": "^0.0.1",
```

Comment:
With 100% difficulty, metamax, the same thing happens every time I download the app, reset the wallet. I will not be able to see the token in the Metamax wallet while it is alive. All tokens are either blocked or locked. The whole system is designed to prevent people from working. Thank you, I am experiencing difficulties, so I wrote, do not take it otherwise. Simplify the system

Reply:
With 100% difficulty, metamax, the same thing happens every time I download the app, reset the wallet. I will not be able to see the token in the Metamax wallet while it is alive. All tokens are either blocked or locked. The whole system is designed to prevent people from working. Thank you, I am experiencing difficulties, so I wrote, do not take it otherwise. Simplify the system

Reply:
These are issues we are working on for PPOM. PPOM code is already code fenced thus as I understand these modules will not be part of final build.